tests/e2e/cucumber/features/smoke/admin-settings/users.ocis.feature

```diff
@@ -102,3 +102,27 @@ Feature: spaces management
       | username    | anna             |
       | displayname | Anna Murphy      |
       | email       | anna@example.org |
+
+
+  Scenario: assigned groups
+    Given "Admin" creates following users
+      | id    |
+      | Alice |
+    And "Admin" creates following groups using API
+      | id       |
+      | sales    |
+      | finance  |
+      | security |
+    And "Admin" adds user to the group using API
+      | user  | group |
+      | Alice | sales |
+    When "Admin" logs in
+    And "Admin" opens the "admin-settings" app
+    And "Admin" navigates to the users management page
+    When "Admin" adds the user "Alice" to the groups "finance,security"
```

Comment:
What do you think about adding `using the sidebar panel` to the step description? We already do that explicitly for other actions, and we have the same for `... using the batch actions`. I think that makes it more clear and consistent.

Reply:
sure, I agree. I'll change it